docs/api/ref/schemas/product_nutrition_v3.yaml

```diff
@@ -0,0 +1,814 @@
+                "prepared_Serving_packaging": 4
+            nutrient_sets:
+              type: array
+              description: |
+                An array of nutrient sets of the product.
+
+                Each nutrient set represents a version of the nutrition facts, defined by a combination of:
+                - the preparation state (e.g. *as sold* or *prepared*),
+                - the reference quantity (*per 100g*, *per 100ml* or *per serving*), and
+                - the source of the data (e.g. *packaging*, *manufacturer*, *estimate*).
+
+                This structure allows capturing multiple nutritional profiles for a single product.
+              items:
+                type: object
+                properties:
+                  preparation:
+                    type: string
+                    enum:
+                      - as sold
+                      - prepared
+                    description: |
+                      Indicates whether the nutrition values refer to the product *as sold* or *prepared*.
+
+                      The preparation state affects nutrient values.
+                  per:
+                    type: string
+                    enum:
+                      - 100g
+                      - 100ml
+                      - serving
+                    description: |
+                      The nutrition data on the package can be per serving, per 100g or per 100ml.
+
+                      This is essential to understand if values in the `nutrients` object apply for a serving, for 100g or for 100ml.
+                  per_quantity:
+                    type: integer
+                    minimum: 0
+                    description: |
+                      The nutrition data on the package can be per serving, per 100g or per 100ml.
+                      When the data is given per serving, the actual quantity that defines one serving may vary 
+                      between products and should be stored in this field.
+  
+                      This is essential to understand to which quantity values in `nutrients` apply for.
+
+                      For example, if the label states "per 250g", then this field should be *250*.
+                    example: 250
+                  per_unit:
+                    type: string
+                    description: |
+                      The nutrition data on the package can be per serving, per 100g or per 100ml.
+                      When the data is given per serving, the actual unit that defines one serving may vary 
+                      between products and should be stored in this field.
+  
+                      This is essential to understand to which quantity values in `nutrients` apply for.
+
+                      For example, if the label states "per 250g", then this field should be *g*.
+                    example: "g"
+                  source:
+                    type: string
+                    enum:
```

Comment:
I'm not sure about the enum there, as some clients could break if we add a new string (e.g. a new database). It may be better to list the possible values in the description, and state that new values may be added.

e.g. an article about how clients could try to handle unexpected valeus in enums in Flutter, but that's a lot of complexity on the client, and each client would need to do it: https://medium.com/@amjadhamdoun0/efficient-enum-handling-in-dart-with-json-converters-5d98dba17c8c